### ISSUE TITLE
VPAT: Geoblacklight -- browse/Search Results

### ISSUE THREAD

**Issue by dlowenberg** [CDL-Dryad/dryad-product-roadmap#1321]
Lots of problems with gem versions.  Lots of stuff now doesn't work at all.

Removed as much crap as I could in order to stop the conflicts with faraday

- [x] had to remove oai and bolognese, does the oai feed and other stuff still work?
- [x] Had to remove wicked_pdf to get server to boot, some say 1.4.0 works instead of 2, but neither worked for me.  Fix
- [x] Had to remove bootsnap in order to get error messages.  Put it back after I'm done.

- [x] Had to comment out many js/css libraries from the discovery area in order to get sprockets not to complain  Get it working again after and re-customize everything about geoblacklight and try to get things to load.  Might be easier if it weren't in an engine.

- [x] Had to remove bootstrap stuff and now icons are gone.  Put back and fix icons.
- [x] Had to comment out many different requirements in gemspec and they need serious cleanup to only require things really necessary.
- [x] CKEditor now doesn't load at all. Fix it.
- [x] Font-awesome is missing everywhere and causing test errors.
---
Issues with geoblacklight not working right after the upgrade.
- [x] Map doesn't show on browse page
- [x] The layout of the "browse" page is mangled.
- [x] The results page
  - [x] Results don't show
  - [x] None go to the right place when clicked
  - [x] Other buttons (such as removing filter) go to home page instead of reloading search
  - [x] Expand accordions don't work 
  - [x] Also Map doesn't show on results page.
  - [x] Facets for limiting search no longer show on up side.
- [x] The 20 recent items on homepage are now broken

---

- [x] Upgrade Geoblacklight
- [x] Move the Search Results h1 to the top of the page.
- [x] Change the facet headings if still necessary
- [x] "One or, ideally, both of: Add a ""skip map"" link equivalent to the ""Skip to main content"" link. Add sensible VO labels to map content (i.e., search here, zoom in, zoom out, and appropriate link text). Add "skip map" or the like? Accept that the map won't be usable with SR, so make navigating around it as easy as possible.

Heading specifications: https://drive.google.com/file/d/1-QyVWCPm8n_VBEswQbmgqsbbCVS-opKI/view
